### ISSUE TITLE
Selecting different POI doesn't change

### ISSUE THREAD

**Issue by thibaultmol** [pietervdvn/MapComplete#2193]
There seems to be a bug in both stable and dev where if you select one object and then click on another object. the sidebar will animate but will stay on the same object it was on first

 can only repro on firefox, not on chromium

firefox 130.0.1  on linux, mapcomplete dev 0.47.4

[Kooha-2024-10-04-09-00-26.webm](https://github.com/user-attachments/assets/be7917a9-0e57-4ca8-999b-1c2969991503)

**Comment by pietervdvn** [comment]
Yeah, I noticed a few times before as well (but didn't come around fixing it).

This is a peculiarity from Svelte in combination with the animations, I hope I fixed it by giving some more time in the slide animation.
